Tools/autotest/common.py

```diff
@@ -11880,7 +11880,7 @@ def tfs_validate_current1(self, value):
         if batt is None:
             raise NotAchievedException("Did not get BATTERY_STATUS message")
         battery_status_current_a = batt.current_battery * 0.01 # cA -> A
-        self.progress("BATTERY_STATUS current==%f frsky==%f" % (battery_status_current_a, current_a))
+        self.progress("BATTERY_STATUS current==%f frsky==%f" % (math.round(battery_status_current_a, 1), math.round(current_a, 1)) # noqa: E501
         if self.compare_number_percent(battery_status_current_a, current_a, 10):
```

Comment:
Rounding needs to happen on this line, not the previous one

Reply:
done